src/random_test.cpp

```diff
@@ -455,6 +457,21 @@ std::string Column::rand_value() {
   return "";
 }
 
+/* prepare single quoted string for LIKE clause */
+std::string Table::prepare_string(int where, std::string str) {
+/* Processing the single quoted values that are returned by 'rand_string' */
+  if (columns_->at(where)->rand_value().at(0) == '\'') {
+    str = columns_->at(where)->rand_value().substr(0,2);
```

Comment:
you can just call rand_value() once. and use it else where. It is heavy operation.

Reply:
Yeah my bad. Idea was to use
if (str.at(0) == '\'')
str=str.substr(0,2)